### ISSUE TITLE
Minor accessibility cleanup

### ISSUE THREAD

**Issue by iefremov** [brave/brave-browser#14176]
Setup proper accessibility info in our UI elements.

For QA: In MacOS accessibility settings enable "Hover text" and "Typing feedback" and check that hovering speedreader and bookmark buttons (with `Cmd` pressed to trigger hover text) works as expected. Also check that creating/closing many tabs quickly works as expected.

**Comment by stephendonner** [comment]
Verified `PASSED` on Brave | 1.24.11 Chromium: 89.0.4389.90 (Official Build) nightly (x86_64)
-- | --
Revision | 62eb262cdaae9ef819aadd778193781455ec7a49-refs/branch-heads/4389@{#1534}
OS | macOS Version 11.2.3 (Build 20D91)

On **macOS Big Sur** (`11.2.3`) I tested:
* the `Hover text` accessibility feature with the Speedreader and Bookmark button UI elements (unable to catch a screenshot of the tooltips, due to timing issue) 
* `Speak typing feedback` 
* quickly opening and closing many tabs

| Hover text | Speak typing feedback |
|-----------|-------------------------|
<img width="780" alt="Screen Shot 2021-03-22 at 3 04 12 PM" src="https://user-images.githubusercontent.com/387249/112064553-65ea4b80-8b20-11eb-9ee2-9f7631dac372.png"> | <img width="780" alt="Screen Shot 2021-03-22 at 3 04 22 PM" src="https://user-images.githubusercontent.com/387249/112064557-6b479600-8b20-11eb-945f-6e35fcf08fd3.png">

Leaving issue unverified, so we (others) can check and verify those two features more fully, on Catalina/Mojave, as well, if (probably) needed 👍
